package.json

```diff
@@ -12,9 +12,12 @@
   ],
   "license": "MIT",
   "main": "./build/angular-gravatar.js",
-  "dependencies": {},
+  "dependencies": {
+    "angular": "^1.2.5"
```

Comment:
The [caret notation](https://docs.npmjs.com/misc/semver#caret-ranges-123-025-004) will automatically make sure it will not install version 2 of the angular package. That being said, Angular 2 will use a different scheme for their packages so there won't be a conflict.